### ISSUE TITLE
Bot aren't help with the releases.

### ISSUE THREAD

**Issue by harshad16** [aicoe-aiops/mailing-list-analysis-toolkit#18]
**Describe the bug**
The Thoth Bot which helps with releases is somehow not responding to the request made by the maintainer. 

**To Reproduce**
create any release issue with maintainer [privilege](https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/blob/18cfa8ef77dd4a69beb318a440ff912d49ee55f0/.thoth.yaml#L21)  

**Expected behavior**
successfully create a changelog, tag, and an image in quay.

**Additional context**
#16 #15

**Comment by harshad16** [comment]
During the release of an application with help of thoth bots, involves [kebechet](https://github.com/thoth-station/kebechet) and [aicoe-ci](https://github.com/AICoE/aicoe-ci) bot application, which works together to release the application.
**kebechet** function to update the version string based on release type major/minor/patch and create a changelog w.r.t to the previous version tag available. 
**aicoe-ci** once a new tag is available, aicoe-ci creates a build based on the details described in the configuration file. 

The `new major release` issue was stuck because kebechet was trying to update the version string and create a changelog. 
During the creation of the changelog, it looked for a previous version tag, as there was no version tag. It started to accumulate all logs from the beginning. The process was blocked, as kebechet function of getting the log, errored out with a fatal issue of `unknown revision`.

Error message: 
```
  cmdline: git log --format=%s --no-merges c71cd82c2339a7b675d7caca79a4a47c4c68a64b
601b8f897354a47470a893a84043fada54c28c9f..HEAD
  stderr: 'fatal: ambiguous argument 'c71cd82c2339a7b675d7caca79a4a47c4c68a64b
601b8f897354a47470a893a84043fada54c28c9f..HEAD': unknown revision or path not in the working tree.
```
there seems to be some commit issue in the git tree. More investigation is required to actually resolve this git tree unknow commit revision issue. 

However, if the maintainer is satisfied with a small changelog, then the hack to resolve this issue to create a previous minor tag 
on an older commit, which in turn would help kebechet create a small changelog and not looking over the whole git tree but a subset. 

This approach was used and @tumido  created a minor tag with an older commit. https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/releases/tag/v0.1.0

The release is fixed with this fix, https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/21
https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/issues/16

**Comment by tumido** [comment]
Thank you @harshad16 for you time and brainpower on this! :1st_place_medal: It is very appreciated! :rocket: 

The git tree issue comes from merging the template repo changes for the first time while not having a previous release it seems. 

Kebechet resolved 2 repository root commits - c71cd82c2339a7b675d7caca79a4a47c4c68a64b (from this repo) and 601b8f897354a47470a893a84043fada54c28c9f (from the template). Which lead to the incorrect command executed by Kebechet:

```
git log --format=%s --no-merges c71cd82c2339a7b675d7caca79a4a47c4c68a64b 601b8f897354a47470a893a84043fada54c28c9f..HEAD
```

only one commit range is allowed in `git log`, Kebechet tried to fetch 2. Probably some variable expansion gone wrong.

**Comment by tumido** [comment]
This issue was resolved via a workaround and bots are helping on release of this repo again. I'm closing this issue now, appropriate bug reports should be tagged with this issue already so the great explanation and root cause documentation doesn't get lost. :slightly_smiling_face:
